Polyfills/Canvas/Source/Canvas.cpp

```diff
@@ -22,15 +22,13 @@ namespace Babylon::Polyfills::Internal
         Napi::Function func = DefineClass(
             env,
             JS_CONSTRUCTOR_NAME,
-            {
-                StaticMethod("loadTTFAsync", &NativeCanvas::LoadTTFAsync),
+            {StaticMethod("loadTTFAsync", &NativeCanvas::LoadTTFAsync),
```

Comment:
What's going on with this? Why only some and not others?

Reply:
Fixed on latest commits.